.sass-lint.yml

```diff
@@ -30,7 +32,7 @@ rules:
   leading-zero:
     - 1
     - include: true
-  max-line-length: 120
+  max-line-length: 0
```

Comment:
This is a boolean option from what I gather. Turning off the line length check is better then it being on and set to 80.